__tests__/app.test.js

```diff
@@ -352,3 +352,135 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?sort_by=:column", () => {
```

Comment:
This describe can be combined with the previous describe for the same base endpoint